neurodamus/core/_engine.py

```diff
@@ -77,23 +77,3 @@ class EngineBase(metaclass=_EngineMeta):
     """A dict of the new connection types and associated Manager class"""
     CircuitPrecedence = 1
```

Comment:
@WeinaJi: I wonder if this precendence should be removed?
I noticed that the `METypeEngine` sets it to 0, but NGV doesn't use it.
It appears to be used to make iteration deterministic:
https://github.com/openbraininstitute/neurodamus/blob/ad08b95c7d69a2d8ed7184dad42b422636464a23/neurodamus/core/configuration.py#L693

However, I think is already done since `__engines` is a dictionary, so it's based on insertion time.
Thoughts?

Reply:
It ensures that the biophysical population get instantiated first in case that they are in a random order in `circuit_config.json`. I think it matters for the load balance (as we are loading only 1 non-virtual population for computing the load balance).

Reply:
does it matter that there may be more than one biophysical population?